app/assets/stylesheets/modules/_client.scss

```diff
@@ -320,3 +320,14 @@ $client-match-action-row-height: 22px;
   background-color: $brand-info;
   border-color: $brand-info;
 }
+
+.client-notes
+{
+}
+
+.client-notes__attribution
+{
+  margin: 0.5em 0 0;
+  font-size: 13px;
+  color: #666;
```

Comment:
Color should come from the variables file, there should be a light gray of some sort there.